packages/twenty-server/src/engine/integrations/custom-code-engine/custom-code-engine.module-factory.ts

```diff
@@ -0,0 +1,56 @@
+import { fromNodeProviderChain } from '@aws-sdk/credential-providers';
+
+import {
+  CustomCodeEngineModuleOptions,
+  CustomCodeEngineDriverType,
+} from 'src/engine/integrations/custom-code-engine/interfaces/custom-code-engine.interface';
+
+import { EnvironmentService } from 'src/engine/integrations/environment/environment.service';
+import { FileStorageService } from 'src/engine/integrations/file-storage/file-storage.service';
+import { FileUploadService } from 'src/engine/core-modules/file/file-upload/services/file-upload.service';
+
+export const customCodeEngineModuleFactory = async (
```

Comment:
yes, we also need this integration layer :)

Reply:
however, there is an issue in the dependencies. We are taking dependency on core-modules/file/file-upload here which is not correct as integrations should be the lowest layer.

Let's also introduce a code-builder in core-modules (that will be able to upload the file to s3 if needed) and call this integration too, we might need to re-think who is responsible of what

Also, we should make sure that we distinguish remote code execution (AWS Lambda vs local runner) from storage location (S3 vs local disk)